api/client/canary/canary.pb.go

```diff
@@ -34,20 +34,20 @@ type Canary struct {
 
 	// Whether the canary service is enabled.
 	Enabled *wrappers.BoolValue `protobuf:"bytes,1,opt,name=enabled,proto3" json:"enabled,omitempty"`
-	// Canary service integrations. You must configure at least one account of
-	// each canary.SupportedType (METRICS_STORE, CONFIGURATION_STORE,
-	// OBJECT_STORE) in order to use Spinnaker's canary service.
+	// Canary service integrations. To use Spinnaker's automated canary service,
```

Comment:
You could even link to this for more information:
https://github.com/spinnaker/kayenta/blob/master/docs/configuring-kayenta.md

Not sure if adding `To use Spinnaker's automated canary service,` makes things clearer to me or more confusing.

There isn't really a canary service.

`To enable Spinnaker's Automated Canary Analysis (ACA) features` makes more sense to me.

ACA is a user experience that spans 4 services (Deck, Deck-Kayenta, Orca, and Kayenta)